app/grandchallenge/evaluation/templates/evaluation/evaluation_list.html

```diff
@@ -101,7 +101,7 @@ <h2>Evaluations</h2>
                             {% endif %}
                         </td>
                         <td>
-                            {% if evaluation.submission.phase.submission_kind == evaluation.submission.phase.SubmissionKind.ALGORITHM %}
+                            {% if evaluation.submission.phase.submission_kind == SubmissionKind.ALGORITHM %}
```

Comment:
Does this actually work? I wouldn't have thought that SubmissionKind would be in the context. Same for the leaderboard row. You can still define `SubmissionKind = SubmissionKind` on the `Phase` model so that all of the existing things still work, then becomes a much smaller changeset.

Reply:
Yeah, the only problem is if those two references somehow shift, but, that is unlikely.